lambdas/question/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionHandler.java

```diff
@@ -278,4 +269,11 @@ private APIGatewayProxyResponseEvent handleException(
     private APIGatewayProxyResponseEvent createNoContentResponse() {
         return new APIGatewayProxyResponseEvent().withStatusCode(HttpStatusCode.NO_CONTENT);
     }
+
+    private void initialServiceRegistry() {
+        this.serviceRegistry.loadKeyStore();
```

Comment:
I think we should move back to doing the keystore loading in the constructor as the keystore and passwd are very unlikely to need changing at runtime